### PR TITLE
Add alphabetical sort to file manager

### PR DESCRIPTION
**Pull request by hackmastera**
Here's my approach; any comments, @tpendragon or @jcoyne ?

Thanks!

### CODE REVIEW

File: app/assets/javascripts/curation_concerns/file_manager/sorting.es6 [hunk 1]
```diff
@@ -82,4 +87,40 @@ export default class SortManager {
       }
     ).toArray()
   }
+
+  get alpha_sort_button() {
+    return $("*[data-action='alpha-sort-action']")
+  }
+
+  initialize_alpha_sort_button() {
+    Blacklight.onLoad(() => {
```
Comment (tpendragon) [inline app/assets/javascripts/curation_concerns/file_manager/sorting.es6]: I don't think the sorting manager gets triggered until Blacklight's loaded, so this shouldn't be necessary.

File: app/assets/javascripts/curation_concerns/file_manager/sorting.es6 [hunk 2]
```diff
@@ -82,4 +87,40 @@ export default class SortManager {
...
+      )
+    })
+    // sort array by title of each object
+    array.sort(function(o1, o2) {
+      let a = o1.title.toLowerCase()
+      let b = o2.title.toLowerCase()
+      return a < b ? -1 : (a > b ? 1 : 0);
+    });
+    // replace contents of #sortable with elements from the array
+    this.element.empty()
```
Comment (tpendragon) [inline app/assets/javascripts/curation_concerns/file_manager/sorting.es6]: Does sorting still work after you re-order this way? I'm not sure how grumpy jquery-ui-sortable gets if you delete its container contents.

Comment (hackmastera) [inline app/assets/javascripts/curation_concerns/file_manager/sorting.es6]: manually confirmed that there's not a problem with this.

File: app/assets/javascripts/curation_concerns/file_manager/sorting.es6 (hunk 1 above)
Comment (hackmastera) [inline app/assets/javascripts/curation_concerns/file_manager/sorting.es6]: Actually you're right and this was causing a bug that I hadn't noticed where the button didn't work unless you reload the page.

**Comment by mjgiarlo** [comment]
:Tada:

**Pull request merged**